dataretrieval/wqp.py

```diff
@@ -659,55 +680,36 @@ def __init__(self, response, **parameters) -> None:
 
         @property
         def site_info(self):
-            if 'sites' in self._parameters:
-                return what_sites(sites=parameters['sites'])
-            elif 'site' in self._parameters:
-                return what_sites(sites=parameters['site'])
-            elif 'site_no' in self._parameters:
-                return what_sites(sites=parameters['site_no'])
-
-
-def _alter_kwargs(kwargs):
-    """Private function to manipulate **kwargs.
-
-    Not all query parameters are currently supported by ``dataretrieval``,
-    so this function is used to set some of them and raise warnings to the
-    user so they are aware of which are being hard-set.
-
-    """
-
-    if kwargs.get('mimeType', 'csv') == 'geojson':
-        warnings.warn('GeoJSON not yet supported, mimeType set to csv.')
-    kwargs['mimeType'] = 'csv'
-
-    return kwargs
-
-
-def _warn_v3_profiles_outage():
-    """Private function for warning message about WQX 3.0 profiles
-    """
-
-    warnings.warn(
-        'USGS discrete water quality data availability '
-        'and format are changing. As of March 2024, '
-        'the data obtained from legacy profiles will not '
-        'include new USGS data or recent updates to existing '
-        'data. To view the status of changes in data '
-        'availability and code functionality, visit: '
-        'https://doi-usgs.github.io/dataRetrieval/articles/Status.html. '
-        'If you have additional questions about these changes, '
-        'email CompTools@usgs.gov.'
+            if "sites" in self._parameters:
+                return what_sites(sites=parameters["sites"])
+            elif "site" in self._parameters:
+                return what_sites(sites=parameters["site"])
+            elif "site_no" in self._parameters:
+                return what_sites(sites=parameters["site_no"])
+
+
+def _check_mimetype(kwargs):
+    mimetype = kwargs.get("mimeType")
+    if mimetype == "geojson":
```

Comment:
This is throwing the `ValueError` when a `mimeType` is not supplied. Suggest maybe adding an if before these ones that says `if mimeType is not None`, or similar.